poolqc/pooling_qc_sample.py

```diff
@@ -6,7 +6,7 @@
 import dash.dependencies as dep
 import dash_table as dt
 import pandas
-import plotly.plotly as plotly
+import chart_studio.plotly as plotly
```

Comment:
This module is never used (neither is `import dash_table.FormatTemplate as FormatTemplate`)